bodhi/server/consumers/masher.py

```diff
@@ -431,7 +431,6 @@ def work(self):
             self.remove_pending_tags()
 
             self._compose_updates()
-
```

Comment:
This change is unrelated to the patch at hand, and I also would prefer to keep this line here.